ralph-wiggum-toolkit/core/scripts/setup-loop.sh

```
@@ -100,13 +103,56 @@ if [[ -n "${PROMPT_FILE_OVERRIDE:-}" ]]; then
-if [[ ! -f "$PROMPT_FILE" ]]; then
-  echo "Error: $PROMPT_FILE not found. Run /ralph init first." >&2
-  exit 1
-fi
-
 # Build the prompt
-PROMPT=$(cat "$PROMPT_FILE")
+V2_BUILD=false
+
+if [[ "$MODE" == "build" ]] && [[ -f "$RALPH_STATE_FILE" ]]; then
+  # v2 build mode: use coordinator prompt and assemble initial task
+  V2_BUILD=true
```

Shellcheck (0.11.0)

[warning] V2_BUILD appears unused. Verify use (or export if used externally).

(SC2034)